src/ch/imedias/rsccfx/localization/Strings.java

```diff
@@ -1,13 +1,12 @@
 package ch.imedias.rsccfx.localization;
 
 /**
- *  Code generated at 11:22:30 25.05.2017
+ *  Code generated at 10:14:06 27.05.2017
```

Comment:
Not quite sure but shouldn't this file be excluded from pushing?
@martinfrancois 
If so, please delete it :)